batteries-included/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM openms/library as library
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /openms-build
+
+RUN apt-get -y update
+RUN apt-get install -y --no-install-recommends --no-install-suggests binutils subversion
+
+RUN for l in lib/*.so; do strip --strip-all $l; done
+
+RUN mkdir /opt/thirdparty && \
+    svn checkout https://github.com/OpenMS/THIRDPARTY/trunk/Linux/64bit /opt/thirdparty && \
+    rm -rf /opt/thirdparty/.svn && \
+    svn checkout https://github.com/OpenMS/THIRDPARTY/trunk/All /opt/thirdparty && \
+    rm -rf /opt/thirdparty/.svn
+
+ENV PATH /opt/thirdparty/Fido:/opt/thirdparty/LuciPHOr2:/opt/thirdparty/MSGFPlus:/opt/thirdparty/MyriMatch:/opt/thirdparty/OMSSA:/opt/thirdparty/XTandem:$PATH
```

Comment:
We have more Thirdparties by now. If it should be up2date with develop, please add Sirius, SpectraST, Comet and Percolator.

Reply:
Ah I see you did that on the final container. Consider doing it here, too.